aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -115,11 +115,16 @@ void remainder_kernel(TensorIterator& iter) {
     AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "remainder_cpu", [&]() {
       cpu_kernel_vec(iter,
         [=](scalar_t a, scalar_t b) __ubsan_ignore_float_divide_by_zero__ -> scalar_t {
-          return a - b * at::native::floor_impl(a / b);
+          scalar_t mod = std::fmod(a, b);
+          if ((mod != 0) && ((b < 0) != (mod < 0))) mod += b;
+          return mod;
         },
         [=](Vec256<scalar_t> a, Vec256<scalar_t> b) {
-          Vec256<scalar_t> r = a - b * (a / b).floor();
-          return r;
+          // TODO: Sleef 3.5 will have a Sleef_remainder{d4,f8}
```

Comment:
If `Sleep_remainder()` will act like IEEE 754-style remainder (e.g. like python [math.remainder()](https://docs.python.org/3/library/math.html#math.remainder) or C++ [std::remainder()](https://en.cppreference.com/w/cpp/numeric/math/remainder)) this will be not behave like the '%' operator or `torch.remainder()`, e.g. 
```
>>> math.remainder(3, 4)
-1.0
>>> 3 % 4
3
```
@peterbell10 I do not know if you had some tricky use of the IEEE 754-style remainder in mind? .. if not I would recommend to remove that reference to `Sleef_remainder`.

Reply:
Yes, this was a misunderstanding. I've removed the comment.